src/main/scala/net/cakesolutions/CakeBuildInfoPlugin.scala

```diff
@@ -75,7 +75,15 @@ object CakeBuildInfoPlugin extends AutoPlugin {
       }
     },
     packageOptions in (Compile, packageBin) +=
-      Package.ManifestAttributes(generalInfo.value.toSeq: _*)
+      Package.ManifestAttributes(generalInfo.value.toSeq: _*),
+    projectRoot := {
+      "git rev-parse HEAD --show-toplevel".!!.split("\n").toSeq match {
+        case Seq(_, topLevel) =>
+          topLevel
+        case _ =>
+          "."
```

Comment:
For scripted tests, we don't operate in a git repo, so its a good idea to always ensure that this setting is defined (i.e. no exception throwing)